File: GSI QA testing tool/UI/UI_125_IdentificationInformation.cs
using GSI_QA_testing_tool.Utilities;
using OpenQA.Selenium;
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Text;
using System.Threading;
using System.Threading.Tasks;

namespace GSI_QA_testing_tool.UI
{
    internal class UI_125_IdentificationInformation
    {
        public static void GoTo(IWebDriver driver)
        {


            if (Data._StopAt.Contains("Identification Information"))
            {
                CustomDialog customDialog = new CustomDialog("Make Selection", "Select pages you want to stop at, make changes and click ok before clicking the Next button");
                customDialog.ShowDialog();
            }




            if (Finder.FindIt(driver, "//label[@for='ctl00_Main_content_ucDrvLicense_rblValidLicense_0']")) 
            {
                Debug.WriteLine("IdentificationInformation is On");
            }
            else
            {
                Debug.WriteLine("IdentificationInformation is Off");
                return;
            }



            Finder.ClickIt(driver, "//label[@for='ctl00_Main_content_ucDrvLicense_rblValidLicense_0']");
            Finder.SendText(driver, "//input[@id='ctl00_Main_content_ucDrvLicense_txtDrvLicenseNumber']", Data._DriverLicense);
            Finder.UseDropDownByIndex(driver, "//select[@id='ctl00_Main_content_ucDrvLicense_ddlStateIssued']", 5);
            Finder.ClickIt(driver, "//input[@id='ctl00_Main_content_btnNext']");

        }
    }
}

File: GSI QA testing tool/UI/UI_125_IdentificationInformation.vb
Imports OpenQA.Selenium
Imports System
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Linq
Imports System.Text
Imports System.Threading
Imports System.Threading.Tasks

Namespace GSI_QA_testing_tool.UI
    Friend Class UI_125_IdentificationInformation
        Public Shared Sub [GoTo](ByVal driver As IWebDriver, ByVal js As IJavaScriptExecutor)
            If Data._StopAt.Contains("Identification Information") Then
                Data.StopAtDialog()
            End If
            If Finder.FindItByCSS(driver, "#ctl00_Main_content_ucDrvLicense_rblValidLicense_1") Then
                Debug.WriteLine("IdentificationInformation is On")
            Else
                Debug.WriteLine("IdentificationInformation is Off")
                Return
            End If

            Thread.Sleep(1000)
            Finder.JSClickIt(driver, "#ctl00_Main_content_ucDrvLicense_rblValidLicense_0", js)
            Finder.SendText(driver, "//input[@id='ctl00_Main_content_ucDrvLicense_txtDrvLicenseNumber']", Data._DriverLicense)
            Finder.UseDropDownByIndex(driver, "//select[@id='ctl00_Main_content_ucDrvLicense_ddlStateIssued']", 5)
            Finder.JSClickIt(driver, "#ctl00_Main_content_btnNext", js)
        End Sub
    End Class
End Namespace
